src/site/stages/build/plugins/create-header-footer.js

```diff
@@ -51,17 +53,22 @@ function createHeaderFooterData(buildOptions) {
       megaMenuData,
     };
 
-    const serialized = JSON.stringify(headerFooter, null, 4);
+    const serialized = JSON.stringify(headerFooter);
 
     const drupalMenu = DRUPALS.PREFIXED_ENVIRONMENTS.has(buildOptions.buildtype)
       ? replaceWithDrupalLinks(headerFooter, files)
       : headerFooter;
-    const drupalMenuSerialized = JSON.stringify(drupalMenu, null, 4);
+    const drupalMenuSerialized = JSON.stringify(drupalMenu);
+
+    const drupalMenuSerializedEscaped = jsesc(drupalMenuSerialized, {
+      json: true,
+      isScriptContext: true,
```

Comment:
Leaving this link here as a breadcrumb:
https://github.com/mathiasbynens/jsesc#isscriptcontext